ci/cloudbuild/scheduled/reap-gke-clusters/reap_gcr_containers.py

```diff
@@ -42,14 +43,16 @@ def protected_image(project_id, image_name):
 
 def delete_images(project_id):
```

Comment:
Should we create a dedicate artifact registry and wipe the whole thing instead of having special protected images?

Reply:
Hmm, my inclination is yes -- let's move our protected images to artifact registry (less risk we'll accidentally delete them) but in general, we have a lot of scripts using google container registry. Let's followup with a separate PR / investigation to see what's involved there if we want to move all of our deployment scripts to best practices. Wdyt?

Reply:
SGTM, I guess I'm not even sure if this part of the script is needed anymore since I do see int-* style artifact registries in the proejct.